concourse/tasks/run-task.sh

```diff
@@ -22,6 +22,12 @@ root_dir=$(pwd)
 : "${COMMAND:?COMMAND not set}"
 : "${CLUSTER:?COMMAND not set}"
 : "${VARIANT:?VARIANT not set}"
+: "${SKIP_DB_MIGRATIONS:?SKIP_DB_MIGRATIONS not set}"
+
+if [[ "$SKIP_DB_MIGRATIONS" == "true" ]]; then
+  echo "Skipping DB Migrations"
+  exit 0
+fi
```

Comment:
The `run-task` job is used by other tasks (e.g. smoke tests, bootstrapping secrets, etc), so we'd need this flag to be checked a bit higher up

e.g. you could create a `run_db_migration.yml` file (a copy of the `run_task.yml` file), and have that first check the flag, then run this file.